source/content/plugins-known-issues.md

```diff
@@ -786,6 +786,16 @@ ___
 
 ___
 
+## PhastPress
+
+<ReviewDate date="2024-08-13" />
+
+**Issue:** The [PhastPress](https://wordpress.org/plugins/phastpress/) plugin creates an SQLLite database on Pantheon's filesystem, which is problematic. Therefore, it is recommended to not use this plugin on Pantheon.
```

Comment:
```suggestion
**Issue:** [PhastPress](https://wordpress.org/plugins/phastpress/) is a site optimization plugin that creates an SQLite database on Pantheon's filesystem. This is problematic as it can vastly increase the amount of filesystem storage your site consumes and run up against resource limits. This plugin is not recommended on Pantheon.
```